examples/named_entity_recognition/ner_wikigold_transformer.ipynb

```diff
@@ -28,8 +28,8 @@
     "\n",
```

Comment:
<p>Maybe add a cell to show an example of sentence list and label list and add this link https://en.wikipedia.org/wiki/Inside%E2%80%93outside%E2%80%93beginning_(tagging) to guide users who need to create their own labeled data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp-recipes/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>